modules/openxBidAdapter.js

```diff
@@ -187,10 +203,25 @@ const OpenxAdapter = function OpenxAdapter() {
       }
     });
 
-    params.callback = 'window.$$PREBID_GLOBAL$$.oxARJResponse';
-    let queryString = buildQueryStringFromParams(params);
+    if (window.XMLHttpRequest && 'withCredentials' in new XMLHttpRequest()) {
+      try {
+        let queryString = buildQueryStringFromParams(params);
+        let url = `//${delDomain}/w/1.0/arj?${queryString}`;
+        useJsonp = false;
+        ajax.ajax(url, $$PREBID_GLOBAL$$.oxARJResponse, void (0), {
+          withCredentials: true
+        });
+      } catch (err) {
+        useJsonp = true;
+        utils.logMessage(`Ajax call failed due to ${err}, sending ARJ via jsonp instead.`);
+      }
+    }
 
-    adloader.loadScript(`//${delDomain}/w/1.0/arj?${queryString}`);
+    if (useJsonp) {
```

Comment:
As mentioned in the previous comment, there are times where we cannot support ajax so we can't really remove jsonp completely. What did you mean by the switching to a scoped callback?